### ISSUE TITLE
About the training epochs

### ISSUE THREAD

**Issue by zmlshiwo** [yzcjtr/GeoNet#43]
Hi,
In your paper, you said training the depth and pose network takes around 30 epochs. Training the residual flow network takes around 200 epochs. I have prepared the data for training using prepare_train_data.py file.  How many iterations are 30 epochs and 200 epochs? I saw the examples in Readme are 350000 iterations and 400000 iterations.
Best,
Zhai

**Comment by yzcjtr** [comment]
Hi @zmlshiwo, I think you can derive the iterations from the number of total training samples multiplied by training epochs.

**Comment by zmlshiwo** [comment]
Ok, thank you for your reply. In your training processing, you made --seq_length=3. Does three pictures count as a training sample? And after I use prepare data file provided by you, for example, for flow, the formatted data does not contain all of KITTI raw dataset sequences, just part of scenes. Is right? We can calculate the training samples using for training depth and pose network, 350000/30=11666?

**Comment by zmlshiwo** [comment]
@yzcjtr And I reproduce the depth results when the training iterations is 150k. How many iterations can I reproduce the flow results?

**Comment by yzcjtr** [comment]
I think it's okay if you follow the scripts I provided for formatting the data. As for the flow experiment, we provided details in the paper and I suggest you adopt a validation set.
